rules/integrations/aws/privilege_escalation_aws_suspicious_saml_activity.toml

```diff
@@ -0,0 +1,71 @@
+[metadata]
+creation_date = "2021/09/22"
+maturity = "production"
+updated_date = "2021/09/22"
+integration = "aws"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when suspicious SAML activity has occurred in AWS. An adversary could gain backdoor access via SAML manipulation.
```

Comment:
Also let's replace "An adversary could gain backdoor access via SAML manipulation." with "An adversary could manipulate SAML to maintain access to the target."